subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -3191,6 +3191,14 @@ static inline void event_len_prep(struct ll_conn *conn)
 		lr->max_rx_octets = sys_cpu_to_le16(LL_LENGTH_OCTETS_RX_MAX);
 		lr->max_tx_octets = sys_cpu_to_le16(conn->default_tx_octets);
 
+		/* Some compilers can't resolve the conditional if/else/else
```

Comment:
Should we instead add a new Kconfig variable or refactor the code?

Reply:
zero initialize in lines 3139 and 3140 instead.

Reply:
> Should we instead add a new Kconfig variable or refactor the code?

I think that would be too much trouble for this problem - I prefer a simple initialization.

Reply:
> zero initialize in lines 3139 and 3140 instead.

-and remove the comment, I presume?

Reply:
yes